packages/anywidget/src/widget.js

```diff
@@ -1,11 +1,11 @@
 // @ts-check
 import { name, version } from "../package.json";
 
-/** @typedef {import("@jupyter-widgets/base").WidgetView} WidgetView */
+/** @typedef {import("@jupyter-widgets/base").WidgetModel} WidgetModel */
 
 /**
  *  @typedef AnyWidgetModule
- *  @prop render {(view: WidgetView) => Promise<undefined | (() => Promise<void>)>}
+ *  @prop render {(view: { model: WidgetModel, el: HTMLElement }) => Promise<undefined | (() => Promise<void>)>}
```

Comment:
Is it odd that the variable is still called `view` here?

Reply:
Ah yes, that's a good suggestion. Maybe a type like `RenderContext` or `RenderProps` would be better (variable names `context` or `props` respectively)